app/controllers/businessmatching/updateservice/RemoveBusinessTypeHelper.scala

```diff
@@ -125,12 +126,11 @@ class RemoveBusinessTypeHelper @Inject()(val authConnector: AuthConnector,
                         (implicit ac: AuthContext, hc: HeaderCarrier, ec: ExecutionContext): OptionT[Future, Seq[ResponsiblePerson]] = {
 
     val emptyActivities = BMBusinessActivities(Set.empty[BMBusinessActivity])
-
     val canRemoveFitProper = (current: Set[BMBusinessActivity], removing: Set[BMBusinessActivity]) => {
       val hasTCSP = current.contains(TrustAndCompanyServices)
       val hasMSB = current.contains(MoneyServiceBusiness)
 
-      (removing.contains(MoneyServiceBusiness) && !hasTCSP) || (removing.contains(TrustAndCompanyServices) && !hasMSB)
+      (removing.contains(MoneyServiceBusiness) && !hasTCSP) || (removing.contains(TrustAndCompanyServices) && !hasMSB) && !ApplicationConfig.phase2ChangesToggle
```

Comment:
Move appConfig toggle call to start of line for readability?

Reply:
Done